docs/cli/data.md

```diff
@@ -0,0 +1,98 @@
+---
+layout: default
+title: data command
+parent: SMT CLI
+nav_order: 2
+---
+
+# Data subcommand
+{: .no_toc }
+
+This subcommand will perform data migration in either the POC or the minimal downtime mode. This subcommand requires users to pass the session file (which contains schema mapping) generated by either the `schema` subcommand or web UI.
+
+{: .highlight }
+The command below assumes that the open-source version of SMT is being used. For the CLI
+reference of the gCloud version of SMT, please refer [here](https://cloud.google.com/sdk/gcloud/reference/alpha/spanner/migration).
+
+<details open markdown="block">
+  <summary>
+    Table of contents
+  </summary>
+  {: .text-delta }
+1. TOC
+{:toc}
+</details>
+## NAME
+
+    ./spanner-migration-tool - migrate data from a source database to
+        Cloud Spanner given a schema
+
+## SYNOPSIS
+
+    ./spanner-migration-tool --session=SESSION --source=SOURCE
+        [--dry-run] [--log-level=LOG_LEVEL] [--prefix=PREFIX]
+        [--skip-foreign-keys] [--source-profile=SOURCE_PROFILE]
+        [--target=TARGET] [--target-profile=TARGET_PROFILE]
+        [--write-limit=WRITE_LIMIT] [GCLOUD_WIDE_FLAG ...]
+
+## DESCRIPTION
+
+    Migrate data from a source database to Cloud Spanner given a
+    schema.
+
+## EXAMPLES
+
+    To copy data to Cloud Spanner given a session file:
+
+        $ ./spanner-migration-tool --session=./session.json \
+            --source=postgresql < ~/cart.pg_dump \
+            --target-profile='instance=spanner-instance'
+
+    To run a minimal downtime data migration:
+
+        $ ./spanner-migration-tool --session=./session.json \
+            --source=MySQL \
+            --source-profile='host=host,port=3306,user=user,password=pwd,dbN\
+        ame=db,streamingCfg=streaming.json' \
+            --target-profile='project=spanner-project,instance=spanner-insta\
+        nce'
+
+## REQUIRED FLAGS
+
+     --session=SESSION
+        Specifies the file that you restore session state from.
+
+     --source=SOURCE
+        Flag for specifying source database (e.g., PostgreSQL, MySQL,
+        DynamoDB).
+
+## OPTIONAL FLAGS
```

Comment:
Also document default behaviour for the optional flags? 

Also whats the target-profile by default?

Reply:
Thats documented in `flags.md`. Added a note to all the files to refer to it.